thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js

```diff
@@ -257,6 +257,19 @@ export default Ember.Controller.extend({
     }
   ),
 
+  hasErrors: Ember.computed(
+    'entitiesService.errors',
+    'timeseriesService.errors',
+    'aggregatesService.errors',
+    'breakdownsService.errors',
+    function () {
```

Comment:
please use 
```
Ember.computed.or('entitiesService.errors', 'timeseriesService.errors','aggregatesService.errors', 'breakdownsService.errors')
```

https://emberjs.com/api/ember/2.17/classes/@ember%2Fobject%2Fcomputed/methods/or?anchor=or

^You may pass in more than two properties and even use property brace expansion. The computed property will return the first truthy value or last falsy value just like JavaScript's || operator

Reply:
fixed